sphinx_ncs_theme/layout.html

```diff
@@ -88,10 +88,18 @@
             data-bs-toggle="dropdown"
             aria-expanded="false"
           >
-            <span class="d-none d-md-inline">nRF Connect SDK</span>
+            <span class="d-none d-md-inline">{{ github_version }}</span>
             <span id="ncsversion" class="d-inline"></span>
           </button>
-          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton"></div>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+            {%- for version in github_versions %}
+            <a
+              class="dropdown-item"
+              href="https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/{{ version }}/"
```

Comment:
If Zoomin is the new platform, why link to GitHub pages?

Reply:
Zoomin will parse the sphinx docs and change it automatically. 

<img width="936" alt="image" src="https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/assets/2525169/9aa4a145-549a-480f-b610-2eef8ab5219f">